yash-datalake/nested/blobTriggerdFunction.json

```diff
@@ -0,0 +1,275 @@
+        "size": "Y1",
+        "family": "Y",
+        "capacity": 0
+      },
+      "kind": "functionapp"
+    },
+    {
+      "type": "Microsoft.Web/sites",
+      "kind": "functionapp",
+      "name": "[parameters('sites_demotriggerapp1234_name')]",
+      "apiVersion": "2018-02-01",
+      "location": "[parameters('location')]",
+      "properties": {
+        "enabled": true,
+        "serverFarmId": "[resourceId('Microsoft.Web/serverfarms', parameters('serverfarms_EastUS2Plan_name'))]",
+        "siteConfig": {
+          "handlerMappings": [
+            {
+              "arguments": "D:\\home\\Python27\\wfastcgi.py",
+              "extension": "fastCgi",
+              "scriptProcessor": "D:\\home\\Python27\\python.exe"
+            }
+          ],
+          "appSettings": [
+            {
+              "name": "AzureWebJobsStorage",
+              "value": "[Concat('DefaultEndpointsProtocol=https;AccountName=',parameters('storage_name'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('storage_name')),'2018-07-01').keys[0].value)]"
+            },
+            {
+              "name": "elasticclusterstorage_STORAGE",
+              "value": "[Concat('DefaultEndpointsProtocol=https;AccountName=',parameters('storage_name'),';AccountKey=',listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('storage_name')),'2018-07-01').keys[0].value)]"
+            },
+            {
+              "name": "esIp",
+              "value": "[parameters('EsIP')]"
+            },
+            {
+              "name": "kibanaFunctionHost",
+              "value": "[parameters('kibanaFunctionHost')]"
+            },
+            {
+              "name": "kibanaFunctionPath",
+              "value": "[parameters('kibanaFunctionPath')]"
+            },
+            {
+              "name": "raweventhubnamespace_RootManageSharedAccessKey_EVENTHUB",
+              "value": "[concat(listkeys(resourceId('Microsoft.EventHub/namespaces/authorizationRules', variables('eventgridEventHubNamespaceName'), 'RootManageSharedAccessKey'), '2017-04-01').primaryConnectionString,';EntityPath=',variables('raweventhub'))]"
+            },
+            {
+              "name": "raweventhub",
+              "value": "[variables('raweventhub')]"
+            }
+          ]
+        },
+        "reserved": false,
+        "scmSiteAlsoStopped": false,
+        "clientAffinityEnabled": false,
+        "clientCertEnabled": false,
+        "hostNamesDisabled": false,
+        "containerSize": 1536,
+        "dailyMemoryTimeQuota": 0
+      },
+      "dependsOn": [
+        "[resourceId('Microsoft.Web/serverfarms', parameters('serverfarms_EastUS2Plan_name'))]",
+        "[concat('Microsoft.EventHub/namespaces/', variables('eventgridEventHubNamespaceName'))]",
+        "[variables('raweventhub')]"
+      ],
+      "resources": [
+        {
+          "name": "[variables('pythonExtension')]",
+          "type": "siteextensions",

```

Comment:
how is this used?

Reply:
This site extension installs python-version 2.7 in the function hosting VM